src/mars.c

```diff
@@ -444,10 +444,12 @@ int tryMain(size_t argc, const char *argv[])
 #elif TARGET_LINUX
     VersionCondition::addPredefinedGlobalIdent("Posix");
     VersionCondition::addPredefinedGlobalIdent("linux");
+    VersionCondition::addPredefinedGlobalIdent("ELFv1");
     global.params.isLinux = true;
 #elif TARGET_OSX
     VersionCondition::addPredefinedGlobalIdent("Posix");
     VersionCondition::addPredefinedGlobalIdent("OSX");
+    VersionCondition::addPredefinedGlobalIdent("ELFv1");
```

Comment:
Just noticed this line in the code today. OS X doesn't use EFL, it uses Mach-O. Or am I missing something?